doc/Doxyfile.in

```diff
@@ -125,7 +125,7 @@ EXAMPLE_PATTERNS       =
 EXAMPLE_RECURSIVE      = NO
 IMAGE_PATH             =
 INPUT_FILTER           =
-FILTER_PATTERNS        =
+FILTER_PATTERNS        = *.py="@FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
```

Comment:
I tried this initially and held the same fear of shebang.
Adding `@PYTHON_EXECUTABLE@` as above:
 `sh: /usr/bin/python2.7 /opt/projects/freeorion/doc/tag_filter.py: No such file or directory`
(both files exist, tag_filter.py is +x)

Note: [make_version.py](https://github.com/freeorion/freeorion/blob/master/cmake/make_versioncpp.py) uses this as well.
I will investigate make_versioncpp.py further in a few hours to see if I missed some CMakeLists factor.